### PR TITLE
[DOC]: Update NymVPN Guides to `v0.0.3`

### PR DESCRIPTION
**Pull request by serinko**
# Description

NymVPN alpha has a [new release](https://github.com/nymtech/nym/releases/tag/nym-vpn-alpha-0.0.3). Upgrade guides and fix flaws. 

- [x] Fix flaws from feedback
- [x] Change version to `v0.0.3`
- [x] Change release page var
- [x] Update sha verification and the example strings
- [x] Ping @tommyv1987 to change vars in his scripts

### CODE REVIEW

File: documentation/dev-portal/src/nymvpn/cli-linux.md [hunk 1]
```diff
@@ -9,17 +9,18 @@ NymVPN is an experimental software and it's for [testing](./testing.md) purposes
 > Any syntax in `<>` brackets is a user's/version unique variable. Exchange with a corresponding name without the `<>` brackets.
 
 1. Open Github [releases page]({{nym_vpn_latest_binary_url}}) and download the binary for Debian based Linux
-2. Open terminal in the same directory and check the the `sha256sum` by running:
+2. Verify sha hash of your downloaded binary with the one listed on the [releases page]({{nym_vpn_latest_binary_url}}). You can use a simple `shasum` command and compare strings (ie with Python) or run in the same directory the following command, exchanging `<SHA_STRING>` with the one of your binary, like in the example:
 ```sh
-sha256sum ./nym-vpn-cli_0.1.0_ubuntu-22.04_amd64.zip
+echo "<SHA_STRING>" | shasum -a 256 -c
+# example:
+echo "0e4abb461e86b2c168577e0294112a3bacd3a24bf8565b49783bfebd9b530e23  nym-vpn-cli_0.1.0_ubuntu-22.04_amd64.zip" | shasum -a 256 -c
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/cli-linux.md:17]: I think you should make it very clear that this is an _example_ hash

File: documentation/dev-portal/src/nymvpn/cli-mac.md [hunk 2]
```diff
@@ -9,21 +9,18 @@ NymVPN is an experimental software and it's for [testing](./testing.md) purposes
...
+2. Verify sha hash of your downloaded binary with the one listed on the [releases page]({{nym_vpn_latest_binary_url}}). You can use a simple `shasum` command and compare strings (ie with Python) or run in the same directory the following command, exchanging `<SHA_STRING>` with the one of your binary, like in the example:
 ```sh
-# x86_64
-sha256sum ./nym-vpn-cli_0.1.0_macos_x86_64.zip
-
-# aarch64
-sha256sum ./nym-vpn-cli_0.1.0_macos_aarch64.zip
+echo "<SHA_STRING>" | shasum -a 256 -c
+# example:
+echo "96623ccc69bc4cc0e4e3e18528b6dae6be69f645d0a592d926a3158ce2d0c269  nym-vpn-cli_0.1.0_macos_x86_64.zip" | shasum -a 256 -c
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/cli-mac.md:17]: see comment on previous sha example

File: documentation/dev-portal/src/nymvpn/gui-linux.md [hunk 3]
```diff
@@ -11,17 +11,18 @@ NymVPN is an experimental software and it's for [testing](./testing.md) purposes
 ### Installation
 
 1. Open Github [releases page]({{nym_vpn_latest_binary_url}}) and download the binary for Debian based Linux
-2. Open terminal in the same directory and check the the `sha256sum` by running:
+2. Verify sha hash of your downloaded binary with the one listed on the [releases page]({{nym_vpn_latest_binary_url}}). You can use a simple `shasum` command and compare strings (ie with Python) or run in the same directory the following command, exchanging `<SHA_STRING>` with the one of your binary, like in the example:
 ```sh
-sha256sum ./nym-vpn-ui_0.0.2_ubuntu-22.04_amd64.zip
+echo "<SHA_STRING>" | shasum -a 256 -c
+# example:
+echo "a5f91f20d587975e30b6a75d3a9e195234cf1269eac278139a5b9c39b039e807  nym-vpn-desktop_0.0.3_ubuntu-22.04_x86_64.zip" | shasum -a 256 -c
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/gui-linux.md:19]: ditto

File: documentation/dev-portal/src/nymvpn/gui-linux.md [hunk 4]
```diff
@@ -11,17 +11,18 @@ NymVPN is an experimental software and it's for [testing](./testing.md) purposes
...
-3. Compare the output with the sha256 hash shared on the [release page]({{nym_vpn_latest_binary_url}})
-4. Extract files with `unzip` command or manually as you are used to
-5. If you prefer to run `.AppImage` make executable by running:
+3. Extract files with `unzip` command or manually as you are used to
+4. If you prefer to run `.AppImage` make executable by running:
 ```sh
 chmod u+x ./appimage/nym-vpn_0.0.2_amd64.AppImage
 ```
-6. If you prefer to use the `.deb` version for installation (Linux only), open terminal in the same directory and run:
+5. If you prefer to use the `.deb` version for installation (Linux only), open terminal in the same directory and run:
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/gui-linux.md]: `.deb` are Debian / Debian-based only right? Maybe make that clear

File: documentation/dev-portal/src/nymvpn/gui-linux.md [hunk 5]
```diff
@@ -33,16 +34,16 @@ sudo apt-get install -f ./nym-vpn_0.0.2_amd64.deb
 
 ### Configuration
 
-7. Create a NymVPN config directory called `nym-vpn` in your `~/.config`, either manually or by a command:
+6. Create a NymVPN config directory called `nym-vpn` in your `~/.config`, either manually or by a command:
 ```sh
 mkdir $HOME/.config/nym-vpn/
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/gui-linux.md:39]: shouldn't this be `mkdir -p`?

File: documentation/dev-portal/src/nymvpn/gui-mac.md [hunk 6]
```diff
@@ -15,38 +15,34 @@ NymVPN is an experimental software and it's for [testing](./testing.md) purposes
...
+3. Verify sha hash of your downloaded binary with the one listed on the [releases page]({{nym_vpn_latest_binary_url}}). You can use a simple `shasum` command and compare strings (ie with Python) or run in the same directory the following command, exchanging `<SHA_STRING>` with the one of your binary, like in the example:
 ```sh
-# aarch64
-sha256sum ./nym-vpn-ui_0.0.2_macos_aarch64.zip
-
-# x86_64
-sha256sum ./nym-vpn-ui_0.0.2_macos_x86_64.zip
+echo "<SHA_STRING>" | shasum -a 256 -c
+# example:
+echo "da4c0bf8e8b52658312d341fa3581954cfcb6efd516d9a448c76d042a454b5df  nym-vpn-desktop_0.0.3_macos_x86_64.zip" | shasum -a 256 -c
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/gui-mac.md:23]: same as previous ones

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: documentation/dev-portal/src/nymvpn/cli-linux.md (hunk 1 above)
Comment (serinko) [reply documentation/dev-portal/src/nymvpn/cli-linux.md:17]: Would you recon to be more clear in definition or drop that example command as a comment as well?

File: documentation/dev-portal/src/nymvpn/gui-linux.md (hunk 5 above)
Comment (serinko) [reply documentation/dev-portal/src/nymvpn/gui-linux.md:39]: doesn't this asume that you may not have `~/.config` at all? We have it without `-p` in the scripts as well and so far no complains, your call .)

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**